bench/workgen/runner/prepare_stress.py

```diff
@@ -167,9 +169,14 @@
 # least run for ~17 minutes.
```

Comment:
transaction is spelt wrong in the above line

Reply:
Fixed 42245c44f0c358f1f8df23b6e702f7bb30304754